Makefile

```diff
@@ -133,43 +133,39 @@ build: generate fmt vet ## Build manager binary.
 
+# by default images expire from quay registry after 14 days
+# set a longer timeout (or set no label to keep images forever)
+LABEL ?= quay.expires-after=14d
 PLATFORM ?= linux/amd64
 # If you wish built the manager image targeting other platforms you can use the --platform flag.
 # (i.e. docker build --platform linux/arm64 ). However, you must enable docker buildKit for it.
 # More info: https://docs.docker.com/develop/develop-images/build_enhancements/
-.PHONY: docker-build
-docker-build: test ## Build docker image with the manager using docker.
-	docker build --platform ${PLATFORM} -t ${IMG} . -f docker/Dockerfile
 
-.PHONY: docker-push
-docker-push: ## Push docker image with the manager using docker.
-	docker push ${IMG}
+.PHONY: operator-build
+operator-build: test ## Build docker image with the manager using docker.
+	$(CONTAINER_ENGINE) build --platform $(PLATFORM) -f docker/Dockerfile -t $(IMG) --label $(LABEL) .
 
-.PHONY: podman-build
-podman-build: test ## Build docker image with the manager using podman.
-	podman build --platform ${PLATFORM} -t ${IMG} . -f docker/Dockerfile
-
-.PHONY: podman-push
-podman-push: ## Push docker image with the manager using podman.
-	podman push ${IMG}
+.PHONY: operator-push
+operator-push: ## Push IMG image to registry
+	$(CONTAINER_ENGINE) push $(IMG)
 
 # PLATFORMS defines the target platforms for  the manager image be build to provide support to multiple
 # architectures. (i.e. make docker-buildx IMG=myregistry/mypoperator:0.0.1). To use this option you need to:
 # - able to use docker buildx . More info: https://docs.docker.com/build/buildx/
 # - have enable BuildKit, More info: https://docs.docker.com/develop/develop-images/build_enhancements/
-# - be able to push the image for your registry (i.e. if you do not inform a valid value via IMG=<myregistry/image:<tag>> than the export will fail)
+# - be able to push the image for your registry (i.e. if you do not inform a valid value via IMG=<myregistry/image:tag> than the export will fail)
 # To properly provided solutions that supports more than one platform you should use this option.
 # If more arches are needed, use comma-separated list: linux/amd64,linux/arm64,linux390x,linux/ppc64le
 PLATFORMS ?= linux/amd64
 .PHONY: docker-buildx
 docker-buildx: test ## Build and push docker image for the manager for cross-platform support
-	# copy existing docker/Dockerfile and insert --platform=${BUILDPLATFORM} into docker/Dockerfile.cross, and preserve the original Dockerfile
-	sed -e '1 s/\(^FROM\)/FROM --platform=\$$\{BUILDPLATFORM\}/; t' -e ' 1,// s//FROM --platform=\$$\{BUILDPLATFORM\}/' docker/Dockerfile > docker/Dockerfile.cross
+	# copy existing docker/Dockerfile and insert --platform=${PLATFORMS} into docker/Dockerfile.cross, and preserve the original Dockerfile
+	sed -e '1 s/\(^FROM\)/FROM --platform=\$$\{PLATFORMS\}/; t' -e ' 1,// s//FROM --platform=\$$\{PLATFORMS\}/' docker/Dockerfile > docker/Dockerfile.cross
```

Comment:
The variable BUILDPLATFORM is an internal/default variable used by docker to build the images with buildx and it should not be replaced by PLATFORMS here.